internal/pkg/earpointer/analysis.go

```diff
@@ -315,7 +349,149 @@ func (vis *visitor) visitSelect(s *ssa.Select) {
 }
 
 func (vis *visitor) visitCall(call *ssa.CallCommon, instr ssa.Instruction) {
-	// TODO: to be added.
+	if builtin, ok := call.Value.(*ssa.Builtin); ok {
+		// special handling of builtin functions
+		vis.visitBuiltin(builtin, instr)
+		return
+	}
+
+	// Collect unification constraints
+	// Here only static calls are supported.
+	// A future extension is to use a more advanced call graph.
+	paramCstrs, retCstrs :=
+		vis.collectCalleeConstraints(call, call.StaticCallee(), instr)
```

Comment:
Consider returning early if `call.StaticCallee()` is `nil`. Without looking at the implementation of `collectCalleeConstraints`, it is not obvious that `paramCstrs` and `retCstrs` will be empty.